src/globals.c

```diff
@@ -43,17 +44,13 @@ hashmap_t *CONSTANTS_MAP;
 type_t *TYPES;
 int types_idx = 0;
 
-ph1_ir_t *GLOBAL_IR;
-int global_ir_idx = 0;
-
-ph1_ir_t *PH1_IR;
-int ph1_ir_idx = 0;
-
 arena_t *INSN_ARENA;
 
 /* BB_ARENA is responsible for basic_block_t / ph2_ir_t allocation */
 arena_t *BB_ARENA;
 
+int dbg_bb_label_idx = 0;
```

Comment:
Avoid using `dbg_` prefix. It is a matter of verbose messages instead of debugging purpose.